pkg/compiler/check.go

```diff
@@ -861,6 +861,7 @@ func (comp *compiler) checkTypeArgs(t *ast.Type, desc *typeDesc, flags checkFlag
 	return args
 }
 
+// TODO: add warning when template arg is not used
```

Comment:
Is this... something else? Or what you just did? :)

Reply:
Forgot to remove =)